src/components/v5/common/ActionSidebar/partials/forms/SplitPaymentForm/partials/SplitPaymentPayoutsTotal/SplitPaymentPayoutsTotal.tsx

```diff
@@ -44,7 +44,7 @@ const SplitPaymentPayoutsTotal: FC<SplitPaymentPayoutsTotalProps> = ({
         'flex items-center gap-3 text-gray-900 text-1',
       )}
     >
-      <Numeral value={summedAmount} decimals={token.decimals} />
+      <Numeral value={summedAmount} decimals={token.decimals} shouldUseDots />
```

Comment:
And use the above defined component here 
`<NumeralWithDots value={summedAmount} decimals={token.decimals} />`